examples/jsm/loaders/TaskManager.js

```diff
@@ -0,0 +1,560 @@
+/**
+ * @author Don McCurdy / https://www.donmccurdy.com
+ * @author Kai Salmen / https://kaisalmen.de
+ */
+
+import { FileLoaderBufferAsync } from "./obj2/utils/FileLoaderBufferAsync.js";
+
+/**
+ *
+ */
+class TaskManager {
+
+    constructor () {
+
+        this.types = new Map();
+        this.verbose = false;
+
+    }
+
+    /**
+     * Set if logging should be verbose
+     *
+     * @param {boolean} verbose
+     * @return {TaskManager}
+     */
+    setVerbose ( verbose ) {
+
+        this.verbose = verbose;
+        return this;
+
+    }
+
+    /**
+     * Returns true if support for the given task type is available.
+     * @param {string} type The type as string
+     * @return boolean
+     */
+    supportsType ( type ) {
```

Comment:
Minor: I think I find the term "task" or "taskType" clearer than just "type" here, and for the methods below.

Reply:
Yes, this is inconsistent.

Reply:
Changed in latest commit.